docs/book/user-guide/starter-guide/manage-artifacts.md

```diff
@@ -102,7 +102,7 @@ you are making a particularly important pipeline run (such as a release) whose
 artifacts you want to distinguish at a glance later.
 
 {% hint style="warning" %}
-Since custom versions cannot be duplicated, the above step can only be run once successfully. To avoid altering your code frequently, consider using a [YAML Config](../advanced-guide/pipelining-features/configure-steps-pipelines.md) for artifact versioning.
+Since custom versions cannot be duplicated, the above step can only be run once successfully. To avoid altering your code frequently, consider using a [YAML Config](../production-guide/configure-pipeline.md) for artifact versioning.
```

Comment:
```suggestion
Since custom versions cannot be duplicated, the above step can only be run once successfully. To avoid altering your code frequently, consider using a [YAML config](../production-guide/configure-pipeline.md) for artifact versioning.
```